.github/workflows/pull_requests.yml

```diff
@@ -27,6 +33,8 @@ jobs:
           configuration-path: .github/labeler-pr-needs-triage.yml
           repo-token: ${{ secrets.GITHUB_TOKEN }}
   SizeLabeler:
+    permissions:
+      pull-requests: write  # for bflad/pr-size-labeler to add labels & comment on PRs
```

Comment:
The `bflad/pr-size-labeler` action is using a custom secret in this case, so we don't need to give it pull request write permissions.

```suggestion
```